.env.development

```diff
@@ -0,0 +1,19 @@
+EXPO_ACCESS_TOKEN="<EXPO_ACCESS_TOKEN>"
```

Comment:
Environment variables that are inserted in DO should not also be in .env files. This is with the extension of the example/local files (because deploying locally on your own machine doesnt not use DO)